include/gutter_tree.h

```diff
@@ -5,50 +5,50 @@
 #include <queue>
 #include <mutex>
 #include <math.h>
-#include "update.h"
+#include "types.h"
 #include "buffer_control_block.h"
-#include "circular_queue.h"
+#include "work_queue.h"
+#include "buffering_system.h"
 
 typedef void insert_ret_t;
 typedef void flush_ret_t;
 typedef std::pair<Node, std::vector<Node>> data_ret_t;
```

Comment:
Unnecessary line: we already define data_ret_t in types.h

Reply:
I'll remove it